book/website/_toc.yml

```
@@ -476,6 +476,17 @@ parts:
         file: ethical-research/activism/activism-case-study-google.md
     - title: Internal Policy Advocacy
       file: ethical-research/internal-policy
+    - title: Self-Reflection
+      file: ethical-research/self-reflection
+      sections: 
+      - title: Identity and Positionality
+        file: ethical-research/self-reflection/sr-positionality
+      - title: Power and Privilege
+        file: ethical-research/self-reflection/sr-power
+      - title: Self-Reflection Prompts
+        file: ethical-research/self-reflection/sr-prompts
+      - title: Further Resources
```

```suggestion
      - title: Resources
```
Just using resources is more consistent with the rest of the book - you can add in all resources in this section.